mico-core/src/main/java/io/github/ust/mico/core/resource/KafkaFaasConnectorDeploymentInfoResource.java

```diff
@@ -0,0 +1,83 @@
+
+import io.github.ust.mico.core.broker.KafkaFaasConnectorDeploymentInfoBroker;
+import io.github.ust.mico.core.dto.response.MicoServiceDeploymentInfoResponseDTO;
+import io.github.ust.mico.core.exception.MicoApplicationNotFoundException;
+import io.github.ust.mico.core.model.MicoService;
+import io.github.ust.mico.core.model.MicoServiceDeploymentInfo;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.hateoas.Resource;
+import org.springframework.hateoas.Resources;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static io.github.ust.mico.core.resource.ApplicationResource.*;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+@Slf4j
+@RestController
+@RequestMapping(value = PATH_APPLICATIONS, produces = MediaTypes.HAL_JSON_VALUE)
+public class KafkaFaasConnectorDeploymentInfoResource {
+
+    public static final String PATH_KAFKA_FAAS_CONNECOTR_DEPLOYMENT_INFORMATION = "kafkaFaasConnectorDeploymentInformation";
+
+    @Autowired
+    private KafkaFaasConnectorDeploymentInfoBroker kafkaFaasConnectorDeploymentInfoBroker;
+
+    @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}/" + PATH_KAFKA_FAAS_CONNECOTR_DEPLOYMENT_INFORMATION)
+    public ResponseEntity<Resources<Resource<MicoServiceDeploymentInfoResponseDTO>>> getKafkaFaasConnectorDeploymentInformation(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
+                                                                                                                                @PathVariable(PATH_VARIABLE_VERSION) String version) {
+        log.debug("Request to getKafkaFaasConnectorDeploymentInformation with the micoApplicationShortname '{}' and version '{}'.", shortName, version);
+        List<MicoServiceDeploymentInfo> micoServiceDeploymentInfos;
+        try {
+            micoServiceDeploymentInfos = kafkaFaasConnectorDeploymentInfoBroker.getKafkaFaasConnectorDeploymentInformation(shortName, version);
+        } catch (MicoApplicationNotFoundException e) {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, e.getMessage());
+        }
+        // Convert to service deployment info DTOs and return it
+        LinkedList<Resource<MicoServiceDeploymentInfoResponseDTO>> micoServiceDeploymentInfoResources = new LinkedList<>();
```

Comment:
We already have the specialized response DTO `KFConnectorDeploymentInfoResponseDTO` that only includes the relevant information. I think we should this instead of `MicoServiceDeploymentInfoResponseDTO`.
@buehlefs What do you think?

Reply:
We should definitely use `KFConnectorDeploymentInfoResponseDTO`.